lib/packages/core/src/api/__tests__/e2e/accountApi.spec.e2e.ts

```diff
@@ -30,6 +32,27 @@ describe(`[E2E] Account Api`, () => {
 
         });
 
+        it('should getAccountTransactions with MultiOut', async () => {
+            const transactionList = await getAccountTransactions(service)(accountId,
```

Comment:
this is ugly. will need to refactor the method somehow. I needed to set includeIndirect to false, to not break with BRS < 2.3.1